packages/wallet/backend/tests/incomingPayment/service.test.ts

```diff
@@ -30,7 +30,7 @@ describe('Incoming Payment Service', () => {
       assetCode: mockedListAssets[0].code,
       assetId: mockedListAssets[0].id,
       assetScale: mockedListAssets[0].scale,
-      virtualAccountId: 'mocked'
+      gateHubUserId: 'mocked'
```

Comment:
Mostly for all the tests, this should be `gateHubWalletId`
```suggestion
      gateHubWalletId: 'mocked'
```